RELEASING.md

```diff
@@ -42,28 +42,51 @@ During the week before release, the Release Champion makes changes in preparatio
 - Update [releaseVersions](https://github.com/adoptium/ci-jenkins-pipelines/blob/master/pipelines/build/regeneration/release_pipeline_generator.groovy#L6) with correct incoming release versions.
 - Update https://github.com/adoptium/mirror-scripts/blob/master/releasePlan.cfg with expected tag, detail see https://github.com/zdtsw/mirror-scripts/tree/issue/3167#skara-repos-and-processes
 
-Create release branch on below repositories:
+#### Branching message for build related repositories
 
+Post the below message to the #build & #release channels in Slack:
+
+In Preparation for next weeks release, I'm proposing to branch the
```

Comment:
Perhaps have this as a single line in backticks to make it clearer which bit to cut & paste?